.github/workflows/CD.yml

```diff
@@ -0,0 +1,64 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: CD
+
+on:
+    push:
+        branches:
+            - '**'
+#   release:
+#     types: [published]
```

Comment:
Currently, the action is triggered on any push to the repository. If setup for github releases this should only be donw upon release/published